src/bp/ui-shared/src/Interface/typings.d.ts

```diff
@@ -21,4 +21,7 @@ declare module 'botpress/shared' {
   export function confirmDialog(message: string, options: ConfirmDialogOptions): Promise<boolean>
   export const style: { TooltipStyle: TooltipStyle }
   export function TreeView<T>(props: TreeViewProps<T>): JSX.Element
+  export function lang(id: string, values?: Record<string, string | PrimitiveType>): string
+  export function langInit()
+  export function langExtend(langs)
```

Comment:
What do you think of moving those in their namespace, eg `intl` so we can keep the main component tree clean?